Rakefile

```diff
@@ -53,6 +55,62 @@ namespace :test do
   end
 end
 
+namespace :ext do
+  rootdir = Pathname('extensions')
+
+  desc 'Build Chrome Extension'
+  task chrome: 'chrome:build'
+
+  namespace :chrome do
+    dist   = 'dist/crx'
+    extdir = rootdir.join(dist)
+    manifest_json = rootdir.join('chrome/manifest.json')
+
+    directory extdir
+
+    task build: [ extdir, 'lib:templates' ] do
+      cd rootdir do
+        cp_r [ 'chrome/html', 'chrome/js', 'chrome/manifest.json', 'img/', 'tmp/lib/' ], dist
```

Comment:
This line ends up copying files which should be ignored (e.g., vim swap files). I will rewrite this task with `git ls-files` or something.